draft-ietf-taps-interface.md

```diff
@@ -768,15 +768,15 @@ This property specifies whether the application needs or prefers to use a transp
 protocol that preserves message boundaries. The default
 is to Prefer Preservation of Message Boundaries.
 
-### Configure Per-Message Reliability {#prop-partially-reliable}
+### Per-Message Reliability {#prop-partially-reliable}
```

Comment:
I think the previous name was better: "Per-Message Reliability" reads as if "True" means that all messages will be reliably transferred. This, however, expresses a desire to be able to *configure* it per message.

Reply:
This was in order to be consistent with all other heading. Isn't you point to for other properties as well?

Reply:
No - e.g. "Reliable Data Transfer (Connection)" means: I'd like to be able to send reliably. It doesn't mean "I want to be able to turn it on/off per-message". "Preservation of Message Boundaries" also just means "I want that, if possible". "Preservation of Data Ordering" means "I'd like in-order" - it doesn't mean "I care about configuring in-order per message". and so forth.